data/collections/nex-gddp-cmip6/nex-gddp-cmip6-crossover.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "nex-gddp-cmip6-crossover",
+    "type": "Collection",
+    "links": [],
+    "title": "CMIP6 2-degree average temperature crossover predictions",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -180,
+                    -90,
+                    180,
+                    90
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2015-01-01T00:00:00Z",
+                    "2101-01-01T00:00:00Z"
+                ]
+            ]
+        }
+    },
+    "license": "MIT",
+    "description": "Predicted year at which the average daily temperature has risen by 2-degrees",
+    "stac_version": "1.0.0",
+    "dashboard:is_periodic": false,
```

Comment:
Should this simply *not* be here? When do we want to add `"dashboard:is_periodic": false`?

Reply:
I think we do want `dashboard:is_periodic` but it should be `true` + we need to add `dashboard:time_density day`. This will tell the summarizer to just return the first and last date and the dashboard UI will create a daily timestep picker. Not sure how nice 30 years of daily data will feel in the UI experience, though, if that's what you mean

Reply:
I'm not sure about this - the rasters in question have *values* which are years rather than, themselves, being indexed by years. In a sense, these two rasters *are* temporal indices. The rasters themselves are definitely not periodic in the normal sense

Reply:
I was thinking of COGs set one year apart. Now that I am looking at the crossing year data I see that there is only one COG for each scenario. Not a time series. Sorry about that. 

Given that, I think the best thing to do is keep `dashboard:is_periodic` `false` and add`dashboard:time_density` `null`.

This will let the dashboard apply the default mosaic xyz logic here but we are definitely pushing the edges of the default cog dashboard display logic here.